### DISCUSSION TITLE
SMA > 150 that is 151, 152 onwards creates a subchart

### DISCUSSION THREAD

**Discussion by premji2k** [kernc/backtesting.py#1166] [Q&A]
Hi ,
I am new to backtesting and started my hands on with this library. I am using the example given and am using SMA  151 and this creates a sub chart
![Screenshot 2024-08-20 175527](https://github.com/user-attachments/assets/3805be24-af66-4bba-a5b1-26a1553b8c72)
Can anyone help me out on this.

I am also looking for non overlay chart to be displayed below as a subplot like RSI, ADX. Any pointers will really help

**Comment by crueltytanya** [comment]
Look at #1045
